controlplane/pkg/monitor/monitor_server.go

```diff
@@ -21,6 +28,11 @@ type Event struct {
 	Entities  map[string]Entity
 }
 
+type Stat struct {
```

Comment:
Can we have a more elaborate name of the structure here? `Stat` is pretty short. Also do we need this type exported?